did-tezos/src/lib.rs

```diff
@@ -511,7 +521,12 @@ impl DIDTz {
                                     anyhow!("Couldn't create JWK from P-256 public key: {}", e)
                                 })?
                             }
-                            p => return Err(anyhow!("{} not supported yet.", p)),
+                            p => {
```

Comment:
```suggestion
                            #[allow(unreachable_patterns)]
                            p => {
```
or `#[allow(unused)]`

to prevent warning when secp256k1 and secp256r1 features are both enabled.

Reply:
Oops I thought I had added it, fixed in 6f4bd0c